### DISCUSSION TITLE
Standard-Installation ohne erfolg

### DISCUSSION THREAD

**Discussion by dennisbausch** [camueller/SmartApplianceEnabler#327] [Installation und Betrieb]
Hallo zusammen, 

mir ist es schon fast peinlich aber ich komme mit der Standard-Installation nicht weiter.  Finde aber den Fehler nicht den ich mache.  

Ich gehe nach Anleitung vor: 
https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Installation_DE.md

- Image auf eine 16 GB SD-Karte geschrieben ( komme auch mit Benutzer pi per ssh drauf)

- USB Stick mit Raspberry PI Imager Formatiert

- Alle drei Dateien direkt  **Ohne Änderungen** ins Hauptverzeichnis des USB Stick runtergeladen.  ( verwende nur LAN)
[install.config](https://raw.githubusercontent.com/camueller/SmartApplianceEnabler/master/install/install.config) 
[install.sh](https://raw.githubusercontent.com/camueller/SmartApplianceEnabler/master/install/install.sh)
[install2.sh](https://raw.githubusercontent.com/camueller/SmartApplianceEnabler/master/install/install2.sh)

- Nach dem der Raspberry gebootet hat habe ich den USB gesteckt es ging das Rote licht aus.

- USB Stick gezogen und der Raspberry  bootet neu.

Ab jetzt passiert allerdings nicht mehr  auch nach 5 Stunden warten.....  auch finde ich keinen Logeintrag in /tmp/install.log

SAE ist nicht installiert und [webmin](https://www.webmin.com/) auch nicht . 

Ich habe es schon mehrfach probiert und komme nicht weiter , kann mir jemand helfen ?

**Comment by camueller** [comment]
Leider läßt sich bei diesem Punkt aus der Ferne nicht sagen, woran es liegt. Bisher lag es immer daran, dass die Dateien nicht unverändert auf die SD-Karte geschrieben wurden. Das wird auch dadurch bestätigt, dass aktuell neue User in der Lage dazu sind, die Installation durchzuführen. 

Du kannst alternativ nach wie vor die [manuelle Installation](https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/ManualInstallation_DE.md) durchführen.

**Reply by dennisbausch** [reply]
Danke für die schnelle Antwort. 

Ich lade die Datei per Safari ( Ziel Datei Speichern unter ) Direkt auf den USB Sick.  Bei einstecken erkennt er den Stick auch und die Rote LED schaltet sich aus. 

Ich verstehe das je selber nicht.  Wenn ich die Installation per ssh anstoßen könnte um ggf. eine eine Fehlermeldung zu posten. 
 
Wäre es denkbar eine Automatische Installation per SSH  zu machen.  In dem ich die Dateien per wget runterlade und ausführe ? 
Leider reichen meine Linux Kenntnisse nicht den Befehl selber zu schreiben. Könnte mir jemand helfen ? 

Danke

**Comment by camueller** [comment]
Nach dem Ziehen des USB-Sticks und dem nachfolgenden Neustart beinnt die eigentliche Installation, deren Fortschritt in der Datei `/tmp/install.log` dokumentiert wird. Wenn diese Datei fehlt, wurde die Installation überhaupt nicht gestartet. Zu beachten ist, dass diese Datei nur bis zum nächsten Reboot vorhanden ist!

Weil Du per ssh bereits auf den Raspberry Pi kommst, kannst Du versuchen, die Installation manuell zu starten. Dazu die folgenden Befehle nacheinander in der Shell ausführen:
```
sudo wget https://raw.githubusercontent.com/camueller/SmartApplianceEnabler/master/install/install2.sh -P /usr/local/bin
sudo wget https://raw.githubusercontent.com/camueller/SmartApplianceEnabler/master/install/install.config -P /usr/local/etc
chmod +x /usr/local/bin/install2.sh
sudo /usr/local/bin/install2.sh
```

**Reply by dennisbausch** [reply]
Danke damit hat es geklappt. 

Da man mit den Raspberry PI Imager ja auch das WLAN beim bespielen des Images  einrichten kann wäre eine  Automatische Installation per ssh mit den von dir aber angegebenen Befehl eine sehr interessante Alternative .  Eventuell kannst du das ja als weitere Variante  mit aufnehmen.  Es würde zumindest eine weitere Fehlerquelle ausschließen. 

Ergänzung noch bei chmod

`
sudo wget https://raw.githubusercontent.com/camueller/SmartApplianceEnabler/master/install/install2.sh -P /usr/local/bin
sudo wget https://raw.githubusercontent.com/camueller/SmartApplianceEnabler/master/install/install.config -P /usr/local/etc
sudo chmod +x /usr/local/bin/install2.sh
sudo /usr/local/bin/install2.sh
`

Danke

**Comment by camueller** [comment]
Ich habe obige Befehle in einem neuen Script zusammengefasst und dokumentiert als weitere Art der Installation:

https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/InstallationViaScript_DE.md

**Reply by dennisbausch** [reply]
Perfekt!   

Danke für die Hilfe
